src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -75,9 +75,12 @@ public static function register($mode = 0)
                     }
                 }
             }
-            $path = realpath($path) ?: $path;
+            $realPath = realpath($path);
+            if (false === $realPath && '-' !== $path) {
```

Comment:
I think `-` has changed on 7.2, the string is different

Reply:
Dang, good catch! I knew this was a bit fragile but don't know what else to use, will have to give it more thought.

Reply:
Thanks to docker I now know this could be "Standard input code"

Reply:
Can't think of a less fragile solution though.